test/test_queue.js

```diff
@@ -1492,6 +1492,50 @@ describe('Queue', function () {
         });
       });
     });
+
+    it.skip('an unlocked job should not be moved to delayed', function(done) {
```

Comment:
ok, they are passing now. Btw I moved back this ```queue.close().finally(done);``` to ```queue.close().then(done, done);``` The problem with finally is that it does not catch errors, it will always call done without arguments.